lib/test-file.js

```diff
@@ -0,0 +1,160 @@
+// Based on
+// https://github.com/smikes/test262-parser/blob/62944f8d000ce7685b917cb7c061301a391a949c/lib/parser.js
+// Copyright (C) 2014, Microsoft Corporation. All rights reserved.
+// This code is governed by the BSD License found in the LICENSE file.
```

Comment:
The referenced LICENSE file differs from our own. I think we'll need a LICENSE-MICROSOFT file...

Reply:
Fixed!